<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>AIxMHC 2025</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
		<link rel="stylesheet" type="text/css" href="./css/style.css">

<!-- v ieee cookie banner v -->
 <script src="https://cmp.osano.com/AzyzptTmRlqVd2LRf/129be117-f9e8-4dad-9b76-292c07870a63/osano.js"></script>
 <link rel="stylesheet" href="https://cookie-consent.ieee.org/ieee-cookie-banner.css" type="text/css"/>
 <!-- ^ ieee cookie banner ^ -->

  </head>
  <body>
		<nav class="navbar navbar-expand-lg bg-body-tertiary">
		  <div class="container-fluid">
		    <a class="navbar-brand" href="#"><img src="./images/logo.png" width="100px"></a>
		    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		      <span class="navbar-toggler-icon"></span>
		    </button>
		    <div class="collapse navbar-collapse" id="navbarSupportedContent">
		      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
		        <li class="nav-item">
		          <a class="nav-link" aria-current="page" href="program.html">Home</a>
		        </li>
		        <li class="nav-item">
		          <a class="nav-link" href="about.html">About AIxMHC</a>
		        </li>
		        <li class="nav-item">
		          <a class="nav-link" href="callforpaper.html">Call For Paper</a>
		        </li>
		        <li class="nav-item dropdown">
				  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Committees</a>
					  <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
						<li><a class="dropdown-item" href="committee.html">Organizing Committee</a></li>
						<li><a class="dropdown-item" href="committee2.html">Program Committee</a></li>
					  </ul>
			      </li>
		        <li class="nav-item">
		          <a class="nav-link active" href="keynotes.html">Keynotes</a>
		        </li>
			<li class="nav-item">
			  <a class="nav-link" href="forum.html">Forum</a>
			</li>
			<li class="nav-item">
			  <a class="nav-link" href="program.html">Program</a>
			</li>
		        <li class="nav-item">
		          <a class="nav-link" href="submission.html">Submission</a>
		        </li>
			<li class="nav-item">
			  <a class="nav-link" href="acceptedpapers.html">Accepted Papers</a>
			</li>
			<li class="nav-item">
		          <a class="nav-link" href="registration.html">Registration</a>
		        </li>
			<li class="nav-item">
		          <a class="nav-link" href="work.html">Workshops</a>
		        </li>
				<li class="nav-item dropdown">
				  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Industry Partners</a>
					  <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
						<li><a class="dropdown-item" href="sponsorship.html">Sponsorship</a></li>
						<li><a class="dropdown-item" href="callfor.html">Call for Industrial Sessions</a></li>
					  </ul>
			      </li>
			    <li class="nav-item dropdown">
				  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Information</a>
					  <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
						<li><a class="dropdown-item" href="venue.html">Conference Venue</a></li>
						<li><a class="dropdown-item" href="hotel.html">Hotel Reservation</a></li>
						<li><a class="dropdown-item" href="taichung.html">Taichung City</a></li>
						<li><a class="dropdown-item" href="https://travel.taichung.gov.tw/en/">Taichung Travel Guide</a></li>
					  </ul>
			    </li>
		        <li class="nav-item">
		          <a class="nav-link" href="contact.html">Contact</a>
		        </li>
		      </ul>
		    </div>
		  </div>
		</nav>
<h3 style="margin:20px">2025 AIxMHC Keynote Speakers</h3>
<div class="KNcontainer">
	<div class="row">
		<div class="left">
			<h4 style="margin:20px">ALIREZA HAGHIGHI, MD, DPhil, FACMG</h4>
			<p>Chief Executive and Director, Harvard International Center for Genetic Disease, Harvard Medical School<br>Founding President of the Advancement Initiative for Medicine and Science<br><br>

				Prof. Alireza Haghighi is the Chief Executive and Director of the Harvard International Center for Genetic Disease and the Founding President of the Advancement Initiative for Medicine and Science (AIMS), a global foundation uniting academics and physicians to advance healthcare and medical science. A pioneering leader at the intersection of life sciences and artificial intelligence, Dr. Haghighi is internationally recognized for spearheading transformative projects that aim to improve global health and well-being. He frequently advises government leaders and industry executives on national health policies, innovation strategies, and precision medicine initiatives.
Dr. Haghighi has received numerous prestigious awards and prizes, including the Global Health Pioneer Award for his innovative strategies and initiatives to enhance the quality of life of people around the world and the International Genetic Disorders Prevention Award.
Dr. Haghighi is an American board-certified geneticist. He completed his graduate and clinical training at the University of Oxford and Harvard Medical School.
<br>
				<br>
				<strong>Title: Genomics and AI: Transforming the Future of Precision Medicine</strong><br>
				<br>
				<strong>Abstract:</strong> The convergence of genomics and artificial intelligence (AI) is revolutionizing the landscape of precision medicine, offering unprecedented opportunities to diagnose, treat, and prevent disease at the individual level. This talk will explore how advances in next-generation sequencing, large-scale biobanks, and AI-driven analytics are enabling deeper insights into human biology, disease mechanisms, and therapeutic responses. By integrating multi-omic data with machine learning models, we can now predict disease risk, tailor treatments to a person’s genetic profile, and accelerate drug discovery. Real-world examples will illustrate how genomic-AI synergies are reshaping clinical care and research. The session will also address the ethical, regulatory, and technical challenges that must be overcome to ensure equitable and responsible deployment of these transformative technologies across global healthcare systems.<br><br>

			</p>
			<br>
		</div>
		<div class="right">
			<img src="images/Speaker1.png" alt="">
			<p><br></p>
		</div>
	</div>
	<div class="divider"></div>
	<div class="row">
		<div class="left">
			<h4 style="margin:20px">Soumya Raychaudhuri, Harvard Medical School, USA</h4>
			<p>
				Soumya Raychaudhuri is an institute member of the Broad Institute and serves as the director for the Center for Data Sciences at Brigham and Women’s Hospital (BWH), where he is also the JS Coblyn MB Brenner Distinguished Chair in Rheumatology/Immunology. Raychaudhuri is a professor of medicine and biomedical informatics at Harvard Medical School. Additionally he is a clinically active rheumatologist and sees patients at Brigham and Women’s Hospital Orthopaedic & Arthritis Center. Raychaudhuri has used genetics, genomics, and human immunology to define the basis of rheumatoid arthritis and other immune-mediated diseases. He is one of the leaders of the International Genetics of Rheumatoid Arthritis consortium, which has identified over 100 risk alleles. He has also been at the forefront of devising statistical and computational methods to localize genetic association signals to causal variants, and to interpret human genetic data in the context of functional information. Raychaudhuri currently has active research programs in the human genetics and functional genomics of tuberculosis, type I diabetes, and rheumatoid arthritis, with a specific focus on using single cell genomic strategies to understand tissue inflammation and the role played by CD4+ T cells and fibroblasts. He is a member of the IGVF, ITN, and AMP AIM consortia. He is the recipient of the Doris Duke Clinical Scientist Development Award and the Henry Kunkel Young Investigator award from the American College of Rheumatology, and he is a member of the American Society for Clinical Investigation and a fellow of the American Association for the Advancement of Science. Raychaudhuri completed his undergraduate degrees in biophysics and mathematics from SUNY Buffalo. He went on to join the Stanford University Medical Scientist Training Program where he completed his M.D. and Ph.D. degrees. He pursued clinical training in internal medicine, followed by subspecialty training in rheumatology at BWH. He concurrently completed postdoctoral training in human genetics at the Broad Institute with Mark Daly. In 2010, he launched his laboratory and joined the faculties of the BWH Divisions of Genetics and Rheumatology.<br>
				<br>
				<strong>Title: Using large scale genomic data to define immune mechanisms of complex autoimmune disease</strong><br>
				<br>
				<strong>Abstract:</strong><br>Rheumatoid arthritis is a canonical autoimmune disease, affecting up to 1% of the population. It is associated with autoantibodies and causes a chronic inflammatory arthritis leading to pain, disability and a reduction in lifespan. To define mechanisms of rheumatoid arthritis, we and others have used multiple strategies, and have sought to define key human mechanisms. Using human genetics, we and others have identified ~150 risk alleles in addition to those within the HLA. To define disease mechanisms, we have used single-cell data and tissue imaging data to define key cell states in the context of tissue inflammation. Finally, we have used functional genomics approaches, including single-cell RNA-seq data, single-cell ATAC-seq data, and gene-editing to define disease mechanisms relevant to these key cell states. Each of these analyses relies heavily on computational methods, including those that use artificial intelligence strategies. Successful definition of key mechanisms may lead to novel therapeutics, and potentially a cure.<br><br>
			</p>
			<br>
		</div>
		<div class="right">
			<img src="images/Speaker2.png" alt="圖片2">
			<p><br></p>
		</div>
	</div>
	<div class="divider"></div>
	<div class="row">
		<div class="left">
			<h4 style="margin:20px">Manolis Kellis, Massachusetts Institute of Technology, USA</h4>
			<p>
				Manolis Kellis is a professor of computer science and artificial intelligence at MIT, and leads  the MIT Computational Biology Group at MIT CSAIL and the Broad Institute of MIT and  Harvard. His research seeks to understand the mechanistic basis of human disease, to  develop new therapeutics that reverse disease circuitry, and to enable personalized medicine, using AI and machine learning to integrate genetics and genomics, single-cell epigenomics and transcriptomics, and high-throughput experiments, applied to Alzheimer's, Obesity, Cancer, Schizophrenia, Cardiovascular, and Immune Disorders. He helped lead several large-scale genomics projects, including Roadmap Epigenomics, ENCODE, Genotype Tissue-Expression (GTEx), and Comparative Genomics projects. He has authored over 280 journal publications cited more than 160,000 times. He received the US Presidential Award for Science and Engineering by Barack Obama, the Mendel Medal for Outstanding Achievements in Science, the NIH Director’s Transformative Research Award, the Argo Science Award by the Hellenic President, the Boston Patent Law Association award, the NSF CAREER award, the Alfred P. Sloan Fellowship, the Technology Review TR35  recognition, the AIT Niki Award, and the Sprowls award for the best Ph.D. thesis in computer science at MIT. He has obtained more than 20 multi-year grants from the NIH, and his trainees hold faculty positions at Stanford, Harvard, CMU, McGill, Johns Hopkins, UCLA, and other top universities. He lived in Greece and France before moving to the US, and he studied and conducted research at MIT, the Xerox Palo Alto Research Center, and the Cold Spring Harbor Lab. For more info, see: <a href="https://compbio.mit.edu/">compbio.mit.edu</a>.<br>
				<br>
				<strong>Title: AI for genomic medicine: disease circuitry, patient heterogeneity, therapeutic design</strong><br>
				<br>
				<strong>Abstract:</strong><br>In this talk I will describe the latest advances in AI for healthcare, medicine, biology, therapeutic design, and pharmaceutical development. We have witnessed a dramatic transformation in AI technologies beyond simply prediction, classification, and correlation to foundational models for understanding biological function, chemical structure, and disease states. In this talk I will describe the foundational technologies that power these advances, and our own work in understanding the mechanistic basis of human disease, using this knowledge to identify new targets and design new therapeutics that reverse disease circuitry, using machine learning to integrate genetics and genomics, single-cell epigenomics and transcriptomics, and high-throughput experiments. I will describe our work in six areas: (1) Genomes, their programming language, their circuitry, epigenomics, dynamics, single-cell multi-omics. (2) Disease mechanism, genetic variation, patient subtyping, personalized medicine, electronic health records. (3) Application to neuroscience, Alzheimer's, schizophrenia, cardiovascular disease, obesity, cancer, evolution. (4) Therapeutic design, drug repurposing, high-throughput experiments, drug screening, genome circuitry manipulation, disease reversal. (5) Statistical genetics, causal inference, geometric deep learning, joint embeddings, contrastive learning, computational chemistry, therapeutic design. (6) Embedding space idea representations, visualization, and navigation for learning, discovery, invention, and collaboration.<br><br>
			</p>
			<br>
		</div>
		<div class="right">
			<img src="images/Speaker3.jpg" alt="圖片3">
			<p><br></p>
		</div>
	</div>
	<div class="divider"></div>
	<div class="row">
		<div class="left">
			<h4 style="margin:20px">Edward Y. Chang, Stanford University, USA</h4>
			<p>
				<strong> Edward Y. Chang</strong>  is an adjunct professor in the Computer Science Department at Stanford University, where he has taught since 2019. He was appointed as co-editor-in-chief of ACM Books, starting in December 2025. He previously served as president of HTC DeepQ Healthcare (2012–2021) and as a director of Research at Google (2006–2012), where he pioneered efforts in scalable machine learning, Web-scale image annotation (2008), and data-centric AI (2010), and also sponsored the ImageNet project. He was a visiting professor at UC Berkeley (2017–2020), working on surgical planning with virtual reality, and earlier he held a tenured faculty position in Electrical and Computer Engineering at UC Santa Barbara (1999–2006). Chang earned his M.S. in Computer Science and Ph.D. in Electrical Engineering from Stanford University. In addition to his technical training, he is well read in philosophy and literature -- an intellectual breadth reflected in his research and writing. He is a Fellow of both ACM and IEEE for his contributions to scalable machine learning and AI in healthcare. His honors include the NSF CAREER Award, Google Innovation Award, the ACM SIGMM Test of Time Award, and the US$1 million XPRIZE for AI-driven disease diagnosis.<br>
				<br>
				<strong>Title: From Generative AI to AGI: Multi-LLM Agent Collaboration as a Path Forward in Healthcare </strong><br>
				<br>
				<strong>Abstract:</strong><br>The rise of large language models (LLMs) has transformed AI, shifting it from passive analysis to generative capabilities, from narrow task-specific tools to general-purpose systems, and from monolithic models to collaborative multi-agent frameworks. Many experts anticipate the emergence of Artificial General Intelligence (AGI) by 2040. Critics argue that LLMs cannot lead to AGI, citing their lack of world models, persistent memory, structured reasoning, and planning capabilities. Critics also highlight how LLMs require massive training data yet still fail to match the efficient few-shot learning demonstrated by even young children.<br>
				<br>This talk challenges these critiques by positioning LLMs not as complete solutions, but as necessary substrates for AGI emergence, analogous to how unconscious processes enable conscious reasoning in humans. Just as humans aren't born with blank slates but with neural priors that facilitate learning, LLMs provide foundational capabilities for in-context learning and environmental adaptation. By augmenting LLMs with transactional reliability, independent-validation mechanisms, Socratic reasoning, and multi-agent architectures, we can address their current limitations. The proposed Multi-LLM Agent Collaboration framework offers a pragmatic, scalable path toward AGI, where intelligence emerges not from a single model but through structured interaction, persistent memory, and collective reasoning across networked systems. Throughout the presentation, precision medicine and personalized healthcare serve as domain examples that can benefit from AGI, demonstrating how multi-agent systems could revolutionize clinical decision support, integrate multi-modal medical data, and accelerate biomedical discovery through collaborative reasoning.<br>
			</p>
			<br>
		</div>
		<div class="right">
			<img src="images/Speaker4.png" alt="圖片4">
			<p><br></p>
		</div>
	</div>
	<div class="divider"></div>
	<div class="row">
		<div class="left">
			<h4 style="margin:20px">Yang C. Fann, National Institutes of Health (NIH), USA</h4>
<p style="color: gray;"><small><strong>* Due to the U.S. government shutdown, Dr. Fann will be attending in a personal capacity.</strong></small></p>
			<p>
				Dr. Fann is Director of the Clinical Informatics Office in the NIH Intramural Research Program, where he leads enterprise IT, bioinformatics, and clinical research informatics initiatives supporting neuroscience and biomedical science. Since joining NINDS in 2002, he has directed bioinformatics and clinical informatics programs, and co-led the Informatics Core for the DoD–NIH Center for Neuroscience and Regenerative Medicine, developing infrastructure for national traumatic brain injury studies. He spearheaded the development of award-winning systems including the AI-assisted procurement tool (POTS) and the Biomedical Research Informatics Computing System (BRICS), and CHIRPI, An LLM-based Enterprise Intelligence Hub Supporting NIH Knowledge Management, earning the inaugural HHS Secretary’s Innovation Award and numerous NIH Director’s Awards. Dr. Fann continues to play a central role in shaping NIH-wide strategies for data infrastructure, clinical informatics, and AI-driven biomedical discovery.<br>
				<br>
				<strong>Title: Connected Data and AI: Driving the Future of Biomedical Discovery</strong><br>
				<br>
				<strong>Abstract:</strong><br>Biomedical research is entering a new era where data-driven insights and advances in artificial intelligence (AI) and the ability to connect diverse data sources are reshaping discovery and translation. At NIH, initiatives such as All of US, CRDC and STRIDES are building a shared clinical informatics ecosystem and infrastructure designed to integrate multimodal data, streamline workflows, and enable secure, scalable access for investigators. By unifying data platforms with AI-driven tools including large language models we can accelerate biomedical innovation to support precision medicine, improve reproducibility, and empower collaborative science across disciplines and institutions. This talk will highlight lessons learned from NIH-wide implementations, demonstrate the critical role of connected data and AI in transforming biomedical research, and explore the opportunities and challenges ahead in realizing their full potential for patient impact.<br>
			</p>
			<br>
		</div>
		<div class="right">
			<img src="images/Speaker5.png" alt="圖片5">
			<p><br></p>
		</div>
	</div>
	<div class="divider"></div>
			<div class="row">
				<div class="left">
					<h4 style="margin:20px">Mark Liao, Academia Sinica, Taiwan</h4>
					<p>
						Mark Liao received his Ph.D degree in electrical engineering from Northwestern University in 1990. In July 1991, he joined the Institute of Information Science, Academia Sinica, Taiwan and currently, is a Distinguished Research Fellow. He has worked in the fields of multimedia signal processing, image processing, computer vision, pattern recognition, video forensics, and multimedia protection for more than 30 years.  During 2009-2011, he was the Division Chair of the computer science and information engineering division II, National Science Council of Taiwan. He is jointly appointed as a Professor of the Department of Electrical Engineering and Computer Science of National Cheng Kung University. During 2009-2012, he was jointly appointed as the Multimedia Information Chair Professor of National Chung Hsing University. Since August 2010, he has been appointed as an Adjunct Chair Professor of Chung Yuan Christian University.  From August 2014 to July 2016, he was appointed as an Honorary Chair Professor of National Sun Yat-sen University.  From November 2016 to November 2019, he was appointed as a Chair Professor of National Chiao-Tung University.  He received the Young Investigators'' Award from Academia Sinica in 1998; the Distinguished Research Award from the National Science Council of Taiwan in 2003, 2010 and 2013; the National Invention Award of Taiwan in 2004; the Distinguished Scholar Research Project Award from National Science Council of Taiwan in 2008; and the Academia Sinica Investigator Award in 2010.  He received the TECO award from the TECO foundation in 2016, and the 64th Academic Award from the Ministry of Education in 2020. His professional activities include:  Co-Chair, 2004 International Conference on Multimedia and Exposition (ICME); Technical Co-chair, 2007 ICME; General Co-Chair, 17th International Conference on Multimedia Modeling; President, Image Processing and Pattern Recognition Society of Taiwan (2006-08); Editorial Board Member, Signal Processing Magazine (2010-13); Associate Editor, Transactions on Image Processing (2009-13), Transactions on Information Forensics and Security (2009-12), Transactions on Multimedia (1998-2001), and ACM Computing Surveys (2018-2021).  Since 2021, he has been a Senior Associate Editor of ACM Computing Surveys.  He has been a Fellow of the since 2013 for contributions to image and video forensics and security.<br>
						<br>
						<strong>
							Title: Video Data-Driven Chronic Pain Recognition in Mice
							<!-- Development of YOLOv4, YOLOv7, and YOLOv9-->
						</strong><br>
						<br>
						<strong>Abstract:</strong><br>Assessing chronic pain behavior in mice is critical for preclinical studies. However, existing methods mostly rely on manual labeling of behavioral features, and humans lack a clear understanding of which behaviors best represent chronic pain. For this reason, existing methods struggle to accurately capture the insidious and persistent behavioral changes in chronic pain. This study proposes a framework to automatically discover features related to chronic pain without relying on human-defined action labels. Our method uses universal action space projector to automatically extract mouse action features, and avoids the potential bias of human labeling by retaining the rich behavioral information in the original video. We also collected a mouse pain behavior dataset, which covers the disease process of mice with neuropathic pain and inflammatory pain on different days. Our method achieves 48.41% accuracy on the pain classification task, which is significantly better than human experts (22.72%) and the popular method B-SOiD (30.52%). Finally, our method revealed differences in drug eﬀicacy for different types of pain on zero-shot Gabapentin drug testing, and the results were consistent with past drug eﬀicacy literature. This study demonstrates the potential clinical application of our method, which can provide new insights into pain research and related drug development.<br><br>
						<!-- YOLOv4 has been ranked first in the world object detection competition for two and a half months. It defeated the R&D teams of international companies such as Google, Amazon, Facebook, Microsoft, and Qualcomm.  The birth of YOLOv4 is actually closely related to a project "Development of Smart Transportation System."
	This project is funded by the Ministry of Science and Technology, and it has led to a four-year cooperative relationship between the Academia Sinica and the listed company Elan Electronics. This keynote speech will explain in detail the beginning and end of the implementation of this smart transportation project, and how to develop YOLOv4, the fastest and most accurate object detector in the world during the execution of the project. In Addition, I will also introduce YOLOv7 and YOLOv9, that were developed by our team in 2022 and early 2024, respectively.  These YOLO series real-time object detection systems developed in Taiwan continue to lead the world stage and strive for supreme glory for Taiwan.-->
					</p>
							<br>
</div>
		<div class="right">
			<img src="images/Speaker6.png" alt="圖片6">
			<p><br></p>
		</div>
	</div> 
</div>
	<div class="background-image"></div>

	<footer>
	  <p></p>
	</footer>

		<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
	  </body>
	</html>
